debian-jessie

```diff
@@ -1,5 +1,10 @@
 FROM debian:jessie
 
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y apt-utils apt-transport-https
+RUN apt-get remove -y resolvconf openresolv network-manager
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y apache2 build-essential curl dnsutils git gnupg iptables iptables-persistent libcurl4-openssl-dev libjpeg-dev libxml2-dev libxslt1-dev mysql-server ntpdate openssh-server openssl postfix postfix-mysql postfix-pcre procmail python-dev python-jinja2 python-lxml python-pgpdump python-pip python-virtualenv rsyslog spambayes sudo tor torsocks wget zlib1g-dev
+
 COPY . /own-mailbox/
 RUN cd /own-mailbox/; \
```

Comment:
> Since I'm very new to this project, it's up to you guys to decide :blush:

Yes, Pierre decides.

About the slow connection, if you can manage a small server on the cloud, you can install docker on it and do the testing there. The download speed on a cloud server is extremely fast. When you are done with testing, you can turn off the server, so that you don't pay much for it. I used to do this a few years ago, when I had a limited connection to the internet.